Kinvey/Kinvey/FindOperation.swift

```diff
@@ -164,7 +164,13 @@ internal class FindOperation<T: Persistable>: ReadOperation<T, [T], Swift.Error>
                         self.executeLocal(completionHandler)
                     }
                 } else {
-                    let entities = [T](JSONArray: jsonArray)
+                    func convert(_ jsonArray: [JsonDictionary]) -> [T] {
+                        let startTime = CFAbsoluteTimeGetCurrent()
+                        let entities = [T](JSONArray: jsonArray)
+                        log.debug("Time elapsed: \(CFAbsoluteTimeGetCurrent() - startTime) s")
```

Comment:
The log will include the file, the line number and the name of the method (even for internal methods like this) where the message was logged